tests/integrations/featureflags/test_featureflags.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+import concurrent.futures as cf
+
+import sentry_sdk
+from sentry_sdk.integrations import _processed_integrations
+from sentry_sdk.integrations.featureflags import FeatureFlagsIntegration
+
+
+def test_featureflags_integration(sentry_init, capture_events):
+    _processed_integrations.discard(
+        FeatureFlagsIntegration.identifier
+    )  # force reinstall
+    sentry_init(integrations=[FeatureFlagsIntegration()])
+    flags_integration = sentry_sdk.get_client().get_integration(FeatureFlagsIntegration)
+
+    flags_integration.set_flag("hello", False)
```

Comment:
Is this the sort of API we want to design?  Our customers will either need to know a lot about the SDK or we will need to educate them.  It might be better to expose a function in the generic feature flags module which hides this behavior.

Reply:
Was trying to be consistent with JS here too. Lucas also noted on that PR this feels kinda jank as an API

Reply:
Let's put a function in the integration module that abstracts this away.

Reply:
Ok yeah definitely. Just remembered the reason we do it like this in JS is bundle size concerns